src/main/flight/pid.c

```diff
@@ -774,6 +775,24 @@ float pidGetAirmodeThrottleOffset(void)
 }
 #endif
 
+static FAST_CODE_NOINLINE void disarmOnImpact(void)
+{
+    // if all sticks are within 5% of center, and throttle low, check acc magnitude for impacts
+    // threshold should be highe enough to avoid unwanted disarms in the air on throttle chops
+
+    // normally the acc calculation would be done only when required, but having them here lets us log the acc magnitude in normal flight
+    float accMagnitude = sqrtf(sq(acc.accADC[Z]) + sq(acc.accADC[X]) + sq(acc.accADC[Y])) * acc.dev.acc_1G_rec - 1.0f;
+    DEBUG_SET(DEBUG_EZLANDING, 7, lrintf(accMagnitude * 10));
+
+    float maxDeflectionAbs = fmaxf(getMaxRcDeflectionAbs(), mixerGetRcThrottle());
```

Comment:
Maybe test `getMaxRcDeflectionAbs(), < .05f &&  mixerGetRcThrottle() < .05f`. They are a bit different values

Reply:
That's better, thanks!
The throttle value could probably be even lower, and they are more easily tweaked if needed later by making the test values independent.